src/datacatalog_tag_manager/tag_datasource_processor.py

```diff
@@ -27,18 +27,43 @@ def create_tags_from_csv(self, file_path):
 
         logging.info('')
         logging.info(f'Creating the Tags...')
-        created_tags = self.__create_tags_from_dataframe(dataframe)
+        created_tags = self.__process_tags_from_dataframe(
+            dataframe, processor=self.__datacatalog_facade.create_or_update_tag)
```

Comment:
I prefer being explicit here, for people that are not used to the code, adding the processor= there, for me improves the code readability.

Reply:
WDYT?

Reply:
Agree, this version LGTM :).